community/lexicon/schema/defs.json

```diff
@@ -0,0 +1,20 @@
+{
+  "lexicon": 1,
+  "id": "community.lexicon.schema.defs",
+  "defs": {
+    "schemaView": {
+      "type": "object",
+      "required": ["uri", "cid", "record", "indexedAt", "dependencies"],
+      "properties": {
+        "uri": { "type": "string", "format": "at-uri" },
+        "cid": { "type": "string", "format": "cid" },
+        "record": { "type": "unknown" },
+        "indexedAt": { "type": "string", "format": "datetime" },
+        "dependencies": {
```

Comment:
Dependency strong refs are returned so that a package manager could resolve a point-in-time snapshot of the lexicon tree. Discussion about that [here](https://github.com/orgs/lexicon-community/discussions/42#discussioncomment-12332562).

The actual CID that the appview returns would be implementation specific and so could either return the CIDs at the time the parent was indexed, the latest CIDs regardless of parent CID that was requested, or something else.